### PR TITLE
parse recent blog posts from RSS feed

### PR DESCRIPTION
**Pull request by benesch**
This should make it easier to keep them up to date via a new shortcut,
`rake blog_posts`.

@jseldess thought this might make your life a little easier.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -83,56 +83,21 @@ CockroachDB is an open source database for building global, scalable cloud servi
...
-        </a>
-    </div>
-</div>
-<div class="row">
-    <div class="col-xs-12">
-        <a href="https://www.cockroachlabs.com/blog/unicode-collation-in-cockroachdb.md/">
-        <div class="blog-post last-entry">
-            <div class="blog-title">Implementing Unicode Collation in CockroachDB</div>
-            <div class="blog-meta">Written by <span class="meta-emphasis">David Eisenstat</span> on <span class="meta-emphasis">Apr 13, 2017</span></div>
+        <a href="{{ post.link }}">
```
Comment (jseldess) [inline index.md]: The links are ending up as `localhost:1313/xxx.html`. Should we do something like this instead?

`<a href="https://www.cockroachlabs.com/blog/{{ post.link }}">`

**Comment by benesch** [comment]
Whoops, the Rakefile was updating `blog-posts.xml` instead of `data/blog_posts.xml`. Should work now!

Now that https://github.com/cockroachlabs/hugo-website/pull/127 has landed, links should be working correctly too.

File: v1.0/index.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ CockroachDB is an open source database for building global, scalable cloud servi
     <div class="col-md-4">
         <div class="roach">
             <a href="start-a-local-cluster.html">
-                <img src="{{ 'images/SCENE_superhero_profile_craig.png' | relative_url }}" alt="Quickstart CockroachDB"/>
+                <img src="images/SCENE_superhero_profile_craig.png" alt="Quickstart CockroachDB"/>
```
Comment (jseldess) [inline v1.0/index.md]: @benesch, these `img` tag changes seem to have caused TC errors...

Comment (benesch) [inline v1.0/index.md]: Whoops, merge skew! Should be fixed now.

**Comment by jseldess** [comment]
LGTM! Works great now. Sorry for the delay.

**Pull request merged**